src/account.ts

```diff
@@ -66,6 +66,21 @@ export abstract class Account {
         calldata?: StringMap,
         options?: InvokeOptions
     ): Promise<InvokeResponse> {
+        let maxFee: starknet.FeeEstimation;
+        if (options?.maxFee && options?.overhead) {
+            const msg = "Both maxFee and overhead cannot be specified";
+            throw new StarknetPluginError(msg);
+        }
+
+        if (!options?.maxFee) {
+            let overhead = options?.overhead || 0.5;
```

Comment:
What if overhead is 0? We should default to 0.5 only if overhead is null or undefined

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/206/commits/253aeb445b361f5e6f81c600166c7f916ef2a545